_topic_map.yml

```diff
@@ -1598,6 +1598,8 @@ Topics:
   File: what-huge-pages-do-and-how-they-are-consumed-by-apps
 - Name: Performance Addon Operator for low latency nodes
   File: cnf-performance-addon-operator-for-low-latency-nodes
+- Name: Provisioning and deploying a Distributed Unit (DU)
```

Comment:
should distributed unit be a proper noun? I think you can keep it in lowercase

Reply:
Approved.